### DISCUSSION TITLE
How stop prisma auto-add relation to table

### DISCUSSION THREAD

**Discussion by mxcdh** [prisma/prisma#18448] [Q&A] [answered]
I have problem with vscode, and prisma:
 - when I remove one relation, prisma after save file added it again.
 - bellow video with problem: https://dsh.re/5bea0

I using PostgreSQL.

**Comment by nurul3101** [comment, marked as answer]
Hello @mxcdh 👋 

Have you installed Prisma VS Code Extension?

It looks like you have enabled format On Save settings in VS Code. If you turn it off, Prisma Extension won't automatically format and add a relation on saving.

**Reply by mxcdh** [reply]
@nurul3101  ok, but this relation is important? 
This extension create me additionally relation, who is not important for me, how its work?

btw
![screenshot-2023-03-23 at 17 01 28](https://user-images.githubusercontent.com/98115983/227263169-5dfd00aa-687a-4080-9cca-a81754018acd.png)
I disabled it, but not helpful, probably it is not this.
I using typeScript with node.

**Reply by nurul3101** [reply]
You are looking for this setting, if you want to disable format on save.
<img width="1211" alt="Screenshot 2023-03-23 at 9 34 21 PM" src="https://user-images.githubusercontent.com/13049676/227263911-d048f535-6baf-40b0-bbe0-4944b0717d7e.png">

**Reply by nurul3101** [reply]
From your video, it looks like the Category model has a relation with PaginationToProduct model. The relation needs to be defined on both sides of the model, so the extension is adding the relation automatically in PaginationToProduct model.

**Reply by mxcdh** [reply]
Thanks!
